bin/array-converter

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env php
+<?php
+
+include(__DIR__ . '/../array-converter.php');
```

Comment:
`include` is a special language construct

    include __DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR  . 'array-converter.php';